### PR TITLE
MetisMenu: add missing types for 2.6.1

### PR DESCRIPTION
**Pull request by denisname**
Add events types, preventDefault properties, and 'dispose' option.

Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
* [Events](https://github.com/onokumus/metisMenu/blob/master/README.md#events)
* [PreventDefault](https://github.com/onokumus/metisMenu/blob/master/README.md#preventdefault)
* [Dispose](https://github.com/onokumus/metisMenu/blob/master/README.md#dispose)
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

File: metismenu/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for metisMenu 2.0.3
+// Type definitions for metisMenu 2.6.1
```
Comment (DanielRosenwasser) [inline metismenu/index.d.ts]: Can you truncate this to 2.6?

File: metismenu/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 // Project: http://github.com/onokumus/metisMenu
 // Definitions by: onokums <https://github.com/onokumus/>
```
Comment (DanielRosenwasser) [inline metismenu/index.d.ts]: Would you be willing to add yourself as an author to this declaration file?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by DanielRosenwasser** [comment]
The declarations look solid. I'll pull these in soon!

Just as a heads up, GitHub isn't recognizing the email address you used for your commits. Feel free to rebase and change the author, or just register the email address with GitHub.

If you don't care, we can just pull the commits in after the feedback is addressed.

**Comment by denisname** [comment]
I didn't understand the problem about the email very well.
Github knows my email, and I got all notifications.

I use GitHub private address:
```
$ git config user.email
denisname@users.noreply.github.com
```
Is this related?

**Comment by denisname** [comment]
I just updated code for MetisMenu 2.6.2:
* The property `doubleTapToGo` was deprecated and is gone in 2.6.2.
* As 2.6.2 is not backward compatible with previous versions, I set back exact version in the header.

**Pull request merged**